gk/main.c

```diff
@@ -1693,6 +1701,15 @@ process_flow_entry(struct flow_entry *fe, struct ipacket *packet,
 	return ret;
 }
 
+static inline void
+prefetch_flow_entry(int i, struct flow_entry *fe_arr[])
+{
+	RTE_BUILD_BUG_ON(sizeof(*fe_arr[0]) <= RTE_CACHE_LINE_SIZE);
+	RTE_BUILD_BUG_ON(sizeof(*fe_arr[0]) > 2 * RTE_CACHE_LINE_SIZE);
```

Comment:
The test above will fail when `RTE_CACHE_LINE_SIZE` is 128. We must test the value of `RTE_CACHE_LINE_SIZE` in order to properly define `prefetch_flow_entry()`. See [example](http://git.dpdk.org/dpdk-stable/tree/lib/librte_eal/common/include/rte_memory.h#n75) of how `rte_memory.h` defines `RTE_CACHE_LINE_SIZE_LOG2` for how to structure the code for this.